gpkitmodels/GP/aircraft/motor/motor_test.py

```diff
@@ -0,0 +1,124 @@
+from gpkit import Model, parse_variables, SignomialsEnabled, SignomialEquality, units
```

Comment:
I think this file needs to be cleaned up.  I like how you've decided to test the model but I want to just clean up this file.